feat(content) Northern Anarchist Map Changes

Sachiho-Vanihalla opened the pull request:
**Content (Artwork / Missions / Jobs)**

## Acknowledgement

- [X] I acknowledge that I have read and understand the [Contributing](https://github.com/endless-sky/endless-sky/blob/master/docs/CONTRIBUTING.md) article.

## To-Do
 - [x] Replace temporary system names with proper, actual star system names.
 - [ ] Rename the "Heimdall" defense fleet to a non-norse name, to avoid stepping on the Deep's toes. Maybe something like Cerberus fleet? A bit on the nose, but also pretty cool!
 - [x] ~~Replace the "Raven (Defender)" placeholder ship with a proper vessel, either bespoke modified version or an existing ship. Headhunters are tempting, but the advanced alloys are too complex to be made by pirates.~~ (This idea has been scrapped)
 - [x] Balance trade prices between all the systems. To the right, the Northern Anarchists should buy food, clothes, medicine, and metals at a higher price. To the left, True Anarchists buy luxury goods, equipment, industrial, electronics, and plastics at a higher price.
 - [x] Add a "Oh you can't escape here" message to Iniquus, just like the Twilight.
 - [x] Give Nemo Station a proper description, ~~as well as completing the mission chain associated with it.~~ (The mission chain will be saved for the later PR involving flavor encounters and the like)
 - [x] Add a single planet to the Tortor system, to give merchant ships an actual reason to fly up here, as currently they're just flying into a dead-end!
 - [x] Add a few uninhabited planets
 - [x] Add a gas giant you can land on, likely the one in Unagi
 - [x] Give the systems their own bespoke anarchist government for plot, missions, and other important stuff.


## Summary
In a similar vein to southern pirate space with the CCOR, this PR aims to refresh and liven up those pirate systems up by the Wanderers! There are three new planets, two new stations (one functional, one abandoned), and three new uninhabited systems. Furthermore, fleets have been updated too, now sporting a mean guard fleet in Almaaz to protect the sole way in, as well as non-combat anarchist ships that pop in every now and then. There's anarchist miners, traders, as well as "corrupt" merchants. They've either paid the bribe, or aren't exactly who they're claiming to be!

These changes serve as the foundation for further expansion two new minor factions up in this part of space. Two anarchist groups, both with a unique take on how anarchy should be run. One specialties in equipment and factories, while the other is textiles and foods. The two factions rely on each other, but that doesn't mean there aren't conflicts, which kicks off a series of events for the player to partake in up there!

Notably, there are a fair few placeholder assets in this PR that need to get sorted out, plus some tweaks here and there I'll likely be doing before the final thing ships, but it's now ready enough for review!

## Screenshots
This is how the systems looked before hand:
![image](https://github.com/user-attachments/assets/0e9667f5-8a27-4ac2-82ef-d0b076eb0bd9)

This is how they look now!
![image](https://github.com/user-attachments/assets/ef15fa84-9318-410c-8842-ec369c77f43c)


## Testing Done
I've gotten really, really distracted while "working" on this pack, so I've been doing one too many "play tests" (just straight up playing the game) to see how everything feels. I've dialed everything in with these tests so they feel pretty appropriate! 

## Save File
This save file can be used to test these changes:
[John Pirate.txt](https://github.com/user-attachments/files/20988643/John.Pirate.txt)

Do note, it uses a unique event that isn't in this PR. Just a simple testing event that changes your swizzle. You'll also notice how in this save, the non-combat ships have the name "Anarchists", yet in others they show up as pirates. If you plan on playing pirate, you'll be able to make this distinction, and I'll see about adding it when a proper pirate start gets added!


## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}

## Wiki Update
 - [ ] PR all this to the wiki once everything is finalized, some stuff is still up in the air!

## Performance Impact
N/A

Sachiho-Vanihalla committed:
feat(content): Changes to Northern Pirate Space for Upcoming Northern Anarchists

Still very very very very early in development, putting here for other members of the pirate project to look at.

TODO:
- Heimdall need a rename and rebalance
- Trade prices need balancing
- Temp systems need proper names and to be fleshed out
- Add expanded outfitters and shipyards to this space
- Get proper art for Masquerade
- Add one more small planet for the pirates

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
Update map planets.txt

Sachiho-Vanihalla committed:
Update map planets.txt

Sachiho-Vanihalla committed:
feat(content): Added Variants

Forgot to add stuff to the variants file, whoops! Also tweaks fleets, need to make some sort of light warship for these guys.

Sachiho-Vanihalla committed:
Merge branch 'Northern-Pirate-Map' of https://github.com/Sachiho-Vanihalla/endless-sky into Northern-Pirate-Map

Sachiho-Vanihalla committed:
Revert "Merge branch 'Northern-Pirate-Map' of https://github.com/Sachiho-Vanihalla/endless-sky into Northern-Pirate-Map"

This reverts commit b086c1c99a244f2f7b135adc60532b9730fa82ea, reversing
changes made to cb426002c0a856e82cbe9c7a28537972f6f3273b.

Sachiho-Vanihalla committed:
feat(content): Placeholder for (still needing to be renamed) Heimdall light warship.

Adds a placeholder, tuned Raven to serve as a LW for the Heimdall fleets. Will likely be replaced in future with something else, just here to get it all functional!

Sachiho-Vanihalla committed:
Update fleets.txt

Sachiho-Vanihalla committed:
feat(content): Monitor variant added to (still need to be renamed) Heimdall fleets/Added small fleet

Adds a new small fleet for the Heimdall, to keep the system populated yet not overwhelmingly so. Furthermore, adds a rare spawn of a Heimdall-refreshed Monitor.

This Monitor has been left MOSTLY alone, on account of just not being worth investing too many resources in. Still, they're reliable ships, so they see the odd use as a new AM boat.

Also, removes the (Warpless) tag from the Heimdall ships, as the warping fleet has now been completely removed. No need for it to stick around!

Sachiho-Vanihalla committed:
feat(content): Tweaked fleet spawn rates and personalities

Makes basic pirate fleets a bit more common, and inversely, makes the mining fleets 50% rarer. Should hopefully make the population feel a bit less choked out by miners.

Also tweaks Heimdall personality so they don't disable, to prevent farming free Republic ships.

Fight them, you cowards.

Sachiho-Vanihalla committed:
feat(content) Cheaper Aeries/Cleaner Almaaz

Makes the Heimdall Aerie a lot cheaper, removing the atomics and missiles. There where WAY too many missiles in play and it was just a mess!

Also removes some of the rocks in Almaaz so the Ravens quit blowing themselves up with heavy rockets, which they do, a LOT.

Sachiho-Vanihalla committed:
Reapply "Merge branch 'Northern-Pirate-Map' of https://github.com/Sachiho-Vanihalla/endless-sky into Northern-Pirate-Map"

This reverts commit 37a35276c4d2baf045778658543b22af39aaca14.

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
feat(content) Adjusted sales for anarchist lore

Adjusts the sales for the anarchist planets, making them better fit the lore of the two factions.

The True Anarchists, being the unorganized and scattered group, focus more on mining and non-combat outfits. They also sell "civilian" ships, like haulers and behemoths.

Meanwhile the Northern Anarchists, with more "organization" and concentrated colonies, plus having a lot of old factories to work with, manufacturer advanced ships and weapons without much of a focus on the more primitive and non-combat sales.

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
feat(content) Updated Unagi

Updated the Unagi system with one new planet, New Montego.

New Montego is a battery manufacturing hub, with a unique ocean that allows for the extraction of lithium and sulfur, plus with abundant phosphorus deposits and a few radioactive isotopes, it's a great world for making power!

Sachiho-Vanihalla committed:
feat(content) New, unnamed, uninhabited systems

Adds three news uninhabited systems to northern space. Obviously they need a final name before the merge, they're here for the playtest.

Also moves Al Hurr and Aya'k'k around a bit to better fit in the new system.

Sachiho-Vanihalla committed:
fix(content) Uploaded missing governments and fleet changes

Final touches before the playtest, making sure I have everything synced.

Uploaded some fleet changes, fleets themselves, and governments that I forgot to upload.

Sachiho-Vanihalla committed:
feat(content) Land images for Masquerade and New Montego

Adds a new image for Masquerade, using a cloudy picture of the yellow mountains in China. Currently missing a high-DPI version though...

Also edits ocean33 to make a new image for New Montego, this one has a high-DPI version too!

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
fix(content) Remove duplicate "New Montego"

Removes a duplicate of the "New Montego" outfitter, whoops!

Sachiho-Vanihalla committed:
Merge branch 'Northern-Pirate-Map' of https://github.com/Sachiho-Vanihalla/endless-sky into Northern-Pirate-Map

Sachiho-Vanihalla committed:
fix(content) Missing items in store

Adds heavy rockets and supercapacitors to two stores, must've missed them somehow!

Sachiho-Vanihalla committed:
fix(chore) Cut down planet descriptions

Cut down the descriptions of two planets so they don't rely on the scrolling text box, keeping the full description saved once that feature finally gets added!

Sachiho-Vanihalla committed:
Update sales.txt

Sachiho-Vanihalla committed:
Update sales.txt

TheGiraffe3 commented:
I'm excited to see this up!

A few notes on the description:

- Cerberus is used in the Unfettered Campaign as a group of pirates frequenting the Misam region; probably unwise to use it here as well.
- Using Headhunters in northern pirate fleets should be fine; a look through `fleets.txt` shows several instances in pirate fleets
- I don't think any of your changes here would require a wiki PR, although I haven't looked through them thoroughly

Sachiho-Vanihalla committed:
Merge branch 'endless-sky:master' into Northern-Pirate-Map

Sachiho-Vanihalla committed:
Added Planet to Tortor

Adds a single backwater planet to the Tortor system, a system which already had a planet in the habitable zone.

It looks roughly the same as the original planet and is mostly there for story content, an industry world well past its prime and in decline.

Sachiho-Vanihalla committed:
Add Iniquus Trapped Message

Adds a trapped message to the Iniquus system, definitely not just stolen straight from the Avgi no sir-

Iniquus isn't actually a black hole by the way. It's a wormhole, it just has an event horizon to prevent you from exiting it. Whoever made this system unable to be left must sure be scared of what's on the other side...

Sachiho-Vanihalla committed:
Adjusting Trade Prices

Adjusts trade prices of the anarchist systems to better reflect their local industries.

The north makes industrial goods, the south makes food and textiles, while Almaaz is an "importer" of medicines and luxury goods.

Sachiho-Vanihalla commented on data/map systems.txt:
```diff
@@ -43615,3 +43743,279 @@ system Zubeneschamali
+system Iniquus
+	pos -390.667 -569.704
+	government Uninhabited
+	attributes "north" "black hole" "notable star" "Iniquus"
```
Good catch!

Sachiho-Vanihalla committed:
Apply suggestions from code review

Removes capitalization from "iniquus" attribute, used for the dialog popup there.

Co-authored-by: Loymdayddaud <145969603+TheGiraffe3@users.noreply.github.com>

Sachiho-Vanihalla committed:
Heimdall Revamp

Totally reworks the Heimdall fleets to completely remove all warships, replacing it with an entirely interceptor and fighter based armada.

The logic behind this is that ultimately, making a bespoke raven variant was a silly idea. The Firebird is too big and expensive to fill this role, the Headhunter too complicated for the local industries to produce in meaningful numbers.

Also when you think about it, the biggest advantage of fighters is that they're VERY cheap ways to put guns in space. So if you never have to leave your home system, why bother with bigger ships? Just make more fighters!

This not only makes the defense fleet more unique, but also more believable in-universe. Fighters and interceptors are cheap to build, doubly so when you've ripped the guts out of a bunch of pillaged merchant ships.

The mentality of giving them all missile based weapons is two-fold. On the one hand it increases survivability as it forces the AI into missile boat mode, but it also makes it so Navy ships are less likely to be disabled. Ultimately giving the player an easy place to farm the Navy seemed like a bad idea, so this helps alleviate that.

Sachiho-Vanihalla committed:
Remove Defender Raven

No reason to keep this around if it's not being used anymore!

Sachiho-Vanihalla committed:
Updated Temp Star System Names

Spend way too long trying to find names for these new star systems, and I have developed a newfound hatred for the paradise worlds for stealing practically every system name I came up with.

Like seriously, they're hoarding the entire Auriga constellation and just flung Almaaz halfway across the galaxy for no reason-

Also removes Al Hurr, since that isn't a thing anymore!

Sachiho-Vanihalla committed:
Add Bespoke Anarchist Governments

In order to handle things like later story missions, custom fleets, job board missions and space port encounters, a custom government for the anarchist factions is needed. This is that!

The idea here is to make them indistinguishable from normal pirates at a glance. However, should you attain positive reputation with them, they will become visible to you since you're now "in the know" so to speak!

Sachiho-Vanihalla committed:
Updated Pirate Job Offer/Destinations

Updates the pirate jobs list to include anarchist planets as valid destination and offer locations for all sorts of stuff!

There are a few exceptions to this rule:
- No "Ruling Warlod" jobs, anarchist mob makes those difficult to establish. Control is more shifted to groups, gangs and mafias n' the like.
- No slavery jobs. While it exists in the north, the overwhelming power of the anti-slavery mob makes it more of a hush-hush practice rather than something blatantly posted on the job board.
- No jobs with a minimum destination 2 or lower, as this would result in jobs staying within that little cluster.
- Jobs that are 4 or lower won't END in anarchist planets for the same reason
- Jobs that don't take place in the north (duh)

Sachiho-Vanihalla committed:
Add Uninhabited Planets

Adds several uninhabited planets (and one gas giant!) to the northern systems, to help make them feel a lot more alive and less like a dead end. A nice little cozy corner akin to Remnant space!

Sachiho-Vanihalla committed:
Change System Governments

Forgot to actually give the custom governments the systems they inhabit, whoops!

Sachiho-Vanihalla commented:
Alright, I said I was going to open this PR for review by the end of February, and while it's _technically_ now March here it's still February somewhere, so I'm counting it as close enough! There is still a **lot** to be done that's for sure, but I think I've gotten everything past the _alpha_ phase and can confidently say we've hit the _beta_ of this whole PR situation. Mostly feature complete, save for a few tweaks and stuff that needs to be filled in, but this should help lay the groundwork!

Key details that should probably be addressed:
- **Corrupt Merchants:** The way I've implemented this is somewhat hacky to say the least. Ideally, an official fix for this would be far better than _yet another_ government added to the files!
- **Pirate Heimdall**: Still gotta come up with a better name for these guys! Maybe something like the F-111, where the name "Flying Pig" was first meant as an insult but later became adopted as the name "Aardvark". Would have to come up with some sort of animal to use... Magpies, maybe?
- **Trade Prices:** It would be nice to have trade prices that match the local economies, but currently things fluctuate _way too much_ and very frequently invert the hierarchy. Some way to tone that down on a system-by-system basis would be great!
- **Fleet Spawns:** I've balanced the spawns of fleets in the area to what _I_ think is good, but we'll have to see what the devs think about my choices there!

So yeah, far from done, but finally at the point where I can say it's ready to get looked over!

Sachiho-Vanihalla committed:
Resolve Conflicts

Removes some HR stuff that I still had on my copy to resolve some conflicts in the back end

Sachiho-Vanihalla committed:
Merge branch 'master' into Northern-Pirate-Map

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6798,118 @@ planet Zydee
+	landscape land/occupas
+	government Uninhabited
+	description `Caribbean, often called "The Gulf of Montego" by the locals, is a calm and temprate gas giant. Monolithic clouds of dense ammonia gas lazily float through the atmosphere, their puffy whites tinted with a heavy teal accent as the sun shines down through the methane-rich atmosphere.`
+	description `	Back during the caravan era, scientific probes of all sort where dropped into the core of Caribbean. Scans from this time revealed the presence of  seemingly solid massive objects deep within the core, far past the pressure rating of any probe avalible. While little research has been done since, it is believed to be the result of temprature inversions, trapped pockets of cold air t ...
```
```suggestion
	description `	Back during the caravan era, scientific probes of all sort were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air that can result in false returns.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6798,118 @@ planet Zydee
+	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the paradise worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corperation, the orbital refinery has stayed in opperation for decades. The entire workfore is volunteers, takin ...
+	description `	The hallways are tight and claustrophobic, hissing machinery and rumbling droans echo through the dimly lit corridors. Despite the huge, deafening climate control units hanging down above you, the temprature inside is enough to make you break a sweat.`
+	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The centeral refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`
+	spaceport `	Overlooking it all is a large balcony, bustling with activity. It's here that patient miners await their refined goods, large bricks of questionably pure metals and packaged plates rolled out on conveyor belts akin to a baggage claim, branded with names and weights. Huge overhead gantrys swing by scarily close to the floor, having little space to move, depositing shipments on small s ...
```
```suggestion
	spaceport `	Overlooking it all is a large balcony, bustling with activity. It's here that patient miners await their refined goods: large bricks of questionably pure metals and packaged plates rolled out on conveyor belts akin to a baggage claim, branded with names and weights. Huge overhead gantries swing by scarily close to the floor, having little space to move, depositing shipments on small self-propelled wagons.`
```

roadrunner56 reviewed (commented):
Doesn't seem like there are a lot of references to the new northern anarchist 'governments' in the planet descriptions of this PR. Is this intentional?

Sachiho-Vanihalla commented on data/map planets.txt:
```diff
@@ -1795,6 +1795,22 @@ planet Farpoint
+	landscape land/badlands9-harro
+	description `Kennecott can best be described as a world well past its prime. The planet was originally colonized for its lackluster but easily accessible natural resources, which where a requirement for the compact, primitive mining machines available at the dawn of the hyperdrive. For a time, Kennecott was a booming center for industrial goods and technological developments. However, the miniat ...
+	description `	While a few businesses remain, mostly providing basic goods and tools, all major industries have long since pulled out of Kennecott. Most locals argue that the Syndicate was responsible for this decline, providing cheaper metals to customers like Betelgeuse at a loss to starve out native competition. The remaining population resents the megacorperation, now chained to this dying wo ...
+	spaceport `The atmosphere of the last remaining space port can best be described as a dead mall. The wide hallways and extensive landing pads carry a lonely aura, once designed to service hundreds of merchant ships at a time. Dozens of aging stores and facilities line the promenade, most now locked behind heavy metal shutters, cobwebs and rust inching up the desolate metal bars. Still, a few loc ...
```
It might be, but I figured it would be the easiest way to convey the kind of _vibe_ the place has without getting to into the nitty-gritty, since I'd imagine most people have a pretty good idea of what a dead mall looks like. If it's too out of place, it's not _crucial_ to the description so we could just axe it if needed!

roadrunner56 replied:
It does feel kind of out of place imo. Dead malls to me feel like a late 20th-early 21st century thing, not a term I'd expect to be relevant 1000 years into the future.

Sachiho-Vanihalla commented on data/map planets.txt:
```diff
@@ -6776,3 +6798,118 @@ planet Zydee
+planet Dot
+	attributes uninhabited
+	landscape land/canyon05a
+	description `While space is limited in the northern pirate kingdom, and Dot is just as inhabitable as any other moon, not a soul has ever set foot on the surface of this dwarf planet with plans on staying. With days as dark as night and a thin atmosphere of near pure nitrogen, the planet has little to offer. Without any strategic importance or valuable materials, Dot is seldom visited by any par ...
```
Ah, this must be a remnant from the original concept I had for a more cohesive, single entity. I'll have to modify this to fit what we have now!

Sachiho-Vanihalla committed:
Many, Many Typo FIxes

Lotta typo fixes. Thanks for catching them, Roadrunner!

Co-authored-by: roadrunner56 <65418682+roadrunner56@users.noreply.github.com>

roadrunner56 commented:
Could you give a summary of what the new governments in this PR are all about, and what they add to the game? It would be helpful to have when reviewing the lore side of this PR.

Sachiho-Vanihalla commented:
> Could you give a summary of what the new governments in this PR are all about, and what they add to the game? It would be helpful to have when reviewing the lore side of this PR.

From a functional perspective, these governments are here primarily to handle things like _specific job board missions_ and _later story content_, which wouldn't be possible if we had simply kept them as "Pirate". By separating them into their own unique things, it adds a lot of flexibility as to _what_ we can do with them story-wise. We can, for example, trigger infighting amongst them, which otherwise wouldn't be possible without a hacky solution.

From a _lore_ perspective, these represent two factions who have different ideas of _how_ anarchy should be run: Taking the idea _literally_ or maintaining the idea _functionally_. Following the fall of the powerful, oppressive baron family that once held these territories, things are still a _massive mess_ now that the people don't have anyone controlling them. This is the result of that. Rather than be stuck in a perpetual squabble with one another, the two sides decided to split the territory in two. This, notably, _excludes_ Almaaz, which functions as just a regular pirate world. This is the primary source of tension between the two, as the matter of who should _"own"_ this system is in constant, heated debate.
 
The True Anarchists are the poster child of anarchy, absolutely no government and just a lawless land. They're the ones who primarily do all of the organic resources such as textiles, foods and the like. They've also got the higher total population and the main resource refinery, but their groups & citizens are a lot more spread out, less cohesive.

The Northern Anarchists (Who I might actually rename to the Northern Anarchist State of Arneb, because the N.A.S.A acronym Timeout suggested was a _really_  funny way to fix the name length issue) practice a _functional_ anarchy, having an extensive council of like 600+ representatives. They basically serve as the guys you go to as an impartial judgement on disputes. They've got all the heavy industry, plus are responsible for putting out bounties on people who infringe on individual rights.

When it comes time to add a story for these guys, it would revolve around this dispute over Almaaz, which comes to a head following an attack on Midas Station (the refinery station) by an unknown assailant. Everyone starts pointing fingers, and if you can't uncover who _actually_ did it, everything's just going to implode in a civil war and destroy _everything_.

Sachiho-Vanihalla committed:
Replace Corrupt Merchant Government

Replaces the corrupt merchant government with the disguised bounty, since that's pretty much does what we want it to do! No need to add more governments if it's not needed, we'll see how this works.

Sachiho-Vanihalla commented:
Turns out several events mess with fleet spawns in the north, so will need to tweak those. I know Free Worlds does it, but are there any other examples we should be worried about?

Sachiho-Vanihalla committed:
Update Campaign Events

Updates the campaign events to account for the new pirate systems.

I took the liberty of converting them to the new "add/remove fleet" system. This fixes a few things, such as a few logistics, paradise and old fleets not being included. It's also just a lot better for future-proofing, since there will be plenty more events in this area!

Sachiho-Vanihalla committed:
Remove "Pirate Traffic" and "Corrupt Merchants"

Removes both the corrupt merchants/pirate sympathizers, as well as the little local traffic fleets from the PR.

Ultimately, while these two would be neat additions to the game, in their current state they're kinda buggy and would probably be best separated into their own PR, as to work properly, they'd likely need new mechanics added alongside of them.

Sachiho-Vanihalla committed:
Update Governments

Replaces those huge lists of "People I HATE" with just a simple, default attitude value. Also, adds the government for the True Anarchists, since that didn't exist yet!

Sachiho-Vanihalla committed:
Rename "Northern Anarchist"

So I ended up going with Timeout's idea of making the name shorter, since it was pretty fun and whimsical to be honest!

Now, this group is known as the Northern Anarchist State of Arneb, otherwise known as N.A.S.A. Rest assured, they aren't related to the defunct space program of days yore, and they'll get VERY annoyed if you mention that to them! (They've heard the joke WAY too many times)

Sachiho-Vanihalla committed:
Update Haven Description

Updates the description of Haven to incorporate the new, full name of the N.A.S.A. Also makes the planet more in-line with a big, (relatively) advanced manufacturing center!

roadrunner56 commented on data/human/human missions.txt:
```diff
@@ -12651,3 +12651,43 @@ mission "A Dead Man's Path 2"
+
+
+
+mission "Trapped in Iniquus"
```
Could you explain the purpose of this mission?

roadrunner56 commented on data/map planets.txt:
```diff
@@ -2433,19 +2452,20 @@ planet Harmony
+		description `Haven was founded by highly successful "privateers" nearly two centuries ago. Since then, following the fall of The Baroness, it has become the capital for the Northern Anarchist State of Arneb. Numerous small manufacturing shops sell outfits that cannot be bought legally, and there is even a small shipyard where customized ships are sold to those who have the right connections.`
+	description `	Most of the population lives in large, urban centers along the few large bodies of fresh water on the planet. The cities themselves are a tightly knit web of roads and towers, surrounded by ancient protective walls and spaceport landing pads. Centuries old high-rises stand shoulder to shoulder with modern skyscrapers, all covered in a mess of dazzling and attention-grabbing hologra ...
+	description `	Much of the planet is desert, but with enough moisture to grow enough crops to feed a large part of the population. Strip mining has turned other parts of the planet into barren wastelands, and has polluted many of the rivers.`
+	spaceport `While a few small landing pads lie inches from the city border, a majority of the spaceport is built inside a large trench that wraps around half of the city. Inside this clautstrophobic trench, a flurry of activitiy kicks up loose trash and debris, large moving platforms pulling ships in and out of hangars built along the walls. Dozens of trucks and trams venture deep into tunnels be ...
```
```suggestion
	spaceport `While a few small landing pads lie inches from the city border, a majority of the spaceport is built inside a large trench that wraps around half of the city. Inside this claustrophobic structure, a flurry of activity kicks up loose trash and debris, as large moving platforms pull ships in and out of hangars built along the walls. Dozens of trucks and trams venture deep into tunnels below, a seedy labyrinth of roads and bunkers built long before the city above.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet "Midas Station"
+	attributes frontier north "north pirate" pirate mining
+	landscape land/loc2
+	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the paradise worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corperation, the orbital refinery has stayed in opperation for decades. The entire workfore is volunteers, takin ...
```
```suggestion
	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the Paradise Worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corporation, the orbital refinery has stayed in operation for decades. The entire workforce is volunteers, taking a small cut of any refined goods to keep the station running.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	landscape land/loc2
+	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the paradise worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corperation, the orbital refinery has stayed in opperation for decades. The entire workfore is volunteers, takin ...
+	description `	The hallways are tight and claustrophobic, with hissing machinery and rumbling drones echoing through the dimly lit corridors. Despite the huge, deafening climate control units hanging down above you, the temperature inside is enough to make you break a sweat.`
+	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The centeral refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`
```
```suggestion
	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The central refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet Rhyac
+	attributes frozen fishing "ice giant" frontier north "north pirate" pirate
+	landscape land/snow9
+	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people. The desperate in need of hiding, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small earth city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mi ...
```
```suggestion
	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people: those desperate for a hiding place, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mirror-like ice in a sparkly shimmer.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet Masquerade
+	attributes farming frontier north "north pirate" pirate
+	landscape land/masquerade
+	description `From orbit, Masquerade looks entirely uninhabbitable. The ozone layer is too thin to block most incoming radiation, leaving the surface nearly dead and incredibly radioactive. However, the planet is home to towering mountains that flank vein-like canyons all across the planet's surface. These sheir cliffs condense Masquerade's humid air into thick, fluffy clouds that hover below the ...
```
```suggestion
	description `From orbit, Masquerade looks entirely uninhabitable. The ozone layer is too thin to block most incoming radiation, leaving the surface nearly dead and incredibly radioactive. However, the planet is home to towering mountains that flank vein-like canyons all across the planet's surface. These sheer cliffs condense Masquerade's humid air into thick, fluffy clouds that hover below the peaks. It's under these clouds that life can thrive, mostly shielded from the harmful gamma radiation above.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	landscape land/snow9
+	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people. The desperate in need of hiding, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small earth city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mi ...
+	description `	Ice fishing is a big business on Rhyac, as most of the land is a jet black volcanic rock, cold and infertile. The locals glide across the frozen ocean on hovercraft, using large harpoon cannons to skewer the gigantic worm-like whales that skirt beneath the ice surface. Most of the inhabitants live in small shacks or hidden bunkers along the equator, with the true population being u ...
+	spaceport `Rhyac’s only public space port sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`
```
```suggestion
	spaceport `Rhyac’s only public spaceport sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people. The desperate in need of hiding, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small earth city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mi ...
+	description `	Ice fishing is a big business on Rhyac, as most of the land is a jet black volcanic rock, cold and infertile. The locals glide across the frozen ocean on hovercraft, using large harpoon cannons to skewer the gigantic worm-like whales that skirt beneath the ice surface. Most of the inhabitants live in small shacks or hidden bunkers along the equator, with the true population being u ...
+	spaceport `Rhyac’s only public space port sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`
+	spaceport `	The main hangar sits nestled inside the shattered remains of an ancient human city ship, catwalks and storefronts bolted along the walls and ceiling of the massive metal structure. A warm orange glow drapes across the room, radiating from heat lamps and low-energy lights. The ramshackle market stalls and bars all huddle around geothermal generators that dot the hangar, spreading a st ...
```
```suggestion
	spaceport `	The main hangar sits nestled inside the shattered remains of an ancient city ship, catwalks and storefronts bolted along the walls and ceiling of the massive metal structure. A warm orange glow drapes the room, radiating from heat lamps and low-energy lights. The ramshackle market stalls and bars all huddle around geothermal generators that dot the hangar, spreading a steamy haze across the floors like a fog machine as they melt the condensation around them.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet Nemo
+	attributes uninhabited
+	landscape land/station50
+	description `The air smells strongly of placeholder.`
```
Needs to be fixed?

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	attributes uninhabited
+	landscape land/mars0
+	government Uninhabited
+	description `Officially, this planet is known simply as Almaaz I. It has poor natural resources, no atmosphere, and is far too close to the sun for any meaningful terraforming. However, a unique combination of a soft, sandy surface and weak gravity leads to Boneyard collecting lots of shipwrecks and debris floating in the Almaaz system. Strong enough to pull wrecks down, but weak enough to not b ...
```
Is the gravity weak or strong?

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	landscape land/mars0
+	government Uninhabited
+	description `Officially, this planet is known simply as Almaaz I. It has poor natural resources, no atmosphere, and is far too close to the sun for any meaningful terraforming. However, a unique combination of a soft, sandy surface and weak gravity leads to Boneyard collecting lots of shipwrecks and debris floating in the Almaaz system. Strong enough to pull wrecks down, but weak enough to not b ...
+	description `	Thousands of wrecks litter the surface of this planet, ranging from tiny chunks of daggers and combat drones to (mostly) whole Navy carriers, all picked clean and left to soak in the sun like skeletons in the desert.`
```
```suggestion
	description `	Thousands of wrecks litter the surface of this planet, ranging from tiny chunks of Daggers and Combat Drones to (mostly) whole Navy Carriers, all picked clean and left to soak in the sun like skeletons in the desert.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	government Uninhabited
+	description `Officially, this planet is known simply as Almaaz I. It has poor natural resources, no atmosphere, and is far too close to the sun for any meaningful terraforming. However, a unique combination of a soft, sandy surface and weak gravity leads to Boneyard collecting lots of shipwrecks and debris floating in the Almaaz system. Strong enough to pull wrecks down, but weak enough to not b ...
+	description `	Thousands of wrecks litter the surface of this planet, ranging from tiny chunks of daggers and combat drones to (mostly) whole Navy carriers, all picked clean and left to soak in the sun like skeletons in the desert.`
+	description `	Interestingly, the planet lacks any sort of water for geologic processes like weathering or erosion to create the sandy surface that covers the planet, a fact which has puzzled scientists. It's believed that Boneyard was subject to heavy orbital bombardment at one point, perhaps from rings that once orbited Almaaz II, but there is little evidence to support this theory.`
```
```suggestion
	description `	Interestingly, the planet lacks any sort of water necessary for geologic processes like weathering or erosion to create the sandy surface that covers the planet, a fact which has puzzled scientists. It's believed that Boneyard was subject to heavy orbital bombardment at one point, perhaps from rings that once orbited Almaaz II, but there is little evidence to support this theory.`
```

roadrunner56 replied:
What's Almaaz II?

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet "New Montego"
+	attributes frontier north "north pirate" pirate factory
+	landscape land/newmontego
+	description `At a glance, New Montego looks like the perfect tropical resort. Beautiful beaches, bright oceans, and a stable climate. However, the strong strong odour in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of ba ...
```
```suggestion
	description `At a glance, New Montego looks like the perfect tropical resort: beautiful beaches, bright oceans, and a stable climate. However, the strong strong odor in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of bacteria that seed the seas.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	landscape land/occupas
+	government Uninhabited
+	description `Caribbean, often called "The Gulf of Montego" by the locals, is a calm and temprate gas giant. Monolithic clouds of dense ammonia gas lazily float through the atmosphere, their puffy whites tinted with a heavy teal accent as the sun shines down through the methane-rich atmosphere.`
+	description `	Back during the caravan era, scientific probes of all sort were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air t ...
```
```suggestion
	description `	Back during the caravan era, scientific probes of all sorts were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air that can result in false returns.`

```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+planet "Anago"
+	attributes frontier north "north pirate" pirate mining
+	landscape land/mars2alt
+	description `Anago is the primary source of raw metals for the nearby New Monetgo outfitters and Haven shipyards, the unusually high metal content and low gravity making it the ideal spot for lunar mining camps. It's believed that Anago is the chunk of a formerly large planet, the outer layers stripped clean to reveal the dense, metallic core.`
```
```suggestion
	description `Anago is the primary source of raw metals for the nearby New Montego outfitters and Haven shipyards, with the unusually high metal content and low gravity making it the ideal spot for lunar mining camps. It's believed that Anago is what remains of a large terrestrial planet, the outer layers stripped clean to reveal the dense, metallic core.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	attributes frontier north "north pirate" pirate factory
+	landscape land/newmontego
+	description `At a glance, New Montego looks like the perfect tropical resort. Beautiful beaches, bright oceans, and a stable climate. However, the strong strong odour in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of ba ...
+	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, beaches accented with bright orange and yellow outcrops that contrast the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their branches a ...
```
```suggestion
	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, with beaches accented with bright orange and yellow outcrops that contrast with the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their canopy a unique mushroom-like look.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	landscape land/newmontego
+	description `At a glance, New Montego looks like the perfect tropical resort. Beautiful beaches, bright oceans, and a stable climate. However, the strong strong odour in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of ba ...
+	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, beaches accented with bright orange and yellow outcrops that contrast the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their branches a ...
+	spaceport `Despite not being safe for swimming, New Montego's oceans have a variety of industrial uses, specifially in the production of batteries. Large industrial pumps leech off the vibrant oceans, excracting and filtering valuable resources from the water through a snaking network of hissing, dripping pipes that weave over and around buildings like vines.`
```
```suggestion
	spaceport `Despite not being safe for swimming, New Montego's oceans have a variety of industrial uses, specifially in the production of batteries. Large industrial pumps leech off of the vibrant oceans, excracting and filtering valuable resources from the water through a snaking network of hissing, dripping pipes that weave over and around buildings like vines.`
```

roadrunner56 commented on data/map planets.txt:
```diff
@@ -6776,3 +6799,118 @@ planet Zydee
+	description `At a glance, New Montego looks like the perfect tropical resort. Beautiful beaches, bright oceans, and a stable climate. However, the strong strong odour in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of ba ...
+	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, beaches accented with bright orange and yellow outcrops that contrast the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their branches a ...
+	spaceport `Despite not being safe for swimming, New Montego's oceans have a variety of industrial uses, specifially in the production of batteries. Large industrial pumps leech off the vibrant oceans, excracting and filtering valuable resources from the water through a snaking network of hissing, dripping pipes that weave over and around buildings like vines.`
+	spaceport `	Spaceports on New Montego are more of a collection of loosely bridged landing pads than a single coherent structure, often nested on top of sturdy outcrops and patches of open land between the trees. Small cargo buggies race along the beaches like a highway, kicking up sand that glistens in the sunlight. Sandstone homes and bleached wood huts huddle up against towering metal structur ...
```
```suggestion
	spaceport `	Spaceports on New Montego are more of a collection of loosely bridged landing pads than a single coherent structure, often nested on top of sturdy outcrops and patches of open land between the trees. Small cargo buggies race along the beaches like a highway, kicking up sand that glistens in the sunlight. Sandstone homes and bleached wood huts huddle up against towering metal structures. Bridges built atop pipes dangle from one platform to the next.`
```

roadrunner56 commented:
Have you tested the descriptions to make sure they don't overflow the UI? Some of them are quite long.
